tools/autograd/gen_variable_type.py

```diff
@@ -759,12 +780,40 @@ def emit_fw_derivatives() -> List[str]:
                                        'formula has been defined for it. This case should only happen for function that '
                                        'take a single TensorList as input. All other cases are not supported right now.')
                 requires_fw_grad = "true"
+
+            content.append(FW_DERIVATIVE_BOOLEAN_TEMPLATE.substitute(
```

Comment:
Nit: Could we inline FW_DERIVATIVE_BOOLEAN_TEMPLATE here? the snippet is pretty small and inlining would improve readability